target/linux/rtl838x/base-files/etc/board.d/02-network

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
```

Comment:
Please stick to the underscore for the file name (02_network)

Reply:
Done.